lua/ulx/modules/sh/cfc_brazil.lua

```diff
@@ -0,0 +1,141 @@
+        offsetY = 3500
+    }, 
+    ["gm_genesis"] = {
+        centerZ = 27600 / 2 - 15300,
+        height = 27600 / 2,
+        fallback = Vector( 6800, 6500, -14974 ),
+        offsetX = 600,
+        offsetY = 600
+    }
+}
+
+local function planTrip( ply )
+    for _ = 1, 20 do
+        local mapData = MAP_POSITION_DATA[game.GetMap()]
+        local pos
+        
+        if mapData then
+            pos = Vector( math.random( MIN_X, MAX_X ), math.random( MIN_Y, MAX_Y ), mapData.centerZ + math.random( -mapData.height, mapData.height ) )
+        else
+            pos = Vector( math.random( MIN_X, MAX_X ), math.random( MIN_Y, MAX_Y ), math.random( MIN_Z, MAX_Z ) )
+        end
+        
+        if util.IsInWorld( pos ) then
+            local minHullSize, maxHullSize = ply:GetCollisionBounds()
+            local minHull = Vector( minHullSize.x - HULL_BUFFER, minHullSize.y - HULL_BUFFER, 0 )
+            local maxHull = Vector( maxHullSize.x + HULL_BUFFER, maxHullSize.y + HULL_BUFFER, 0.001 )
+            
+            local validationTrace = util.TraceHull( {
+                start = pos,
+                endpos = pos + Vector( 0, 0, maxHullSize.z ),
+                mins = minHull,
+                maxs = maxHull
+            } )
+            
+            if not validationTrace["Hit"] then
+                local floorTrace = util.TraceHull( {
+                    start = pos,
+                    endpos = pos + Vector( 0, 0, MIN_Z ),
+                    mins = minHull,
+                    maxs = maxHull
+                } )
+                
+                return floorTrace["HitPos"] + Vector( 0, 0, 1 )
+            end
+        end
+    end
+    
+    if not mapData then return Vector( 0, 0, 0 ) end
+    
+    local fallback = mapData.fallback
+    local offsetX = math.random( -mapData.offsetX, mapData.offsetX )
+    local offsetY = math.random( -mapData.offsetY, mapData.offsetY )
+    
+    return fallback + Vector( offsetX, offsetY, 0 )
+end
+
+local function sendToPos( caller, targets, message )
+    for k, v in pairs( targets ) do
+        v:SetPos( planTrip( v ) )
+    end
+    
+    ulx.fancyLogAdmin( caller, message, targets )
+end
+
+local function runBrazil( caller, targets )
+    sendToPos( caller, targets, "#A sent #T to Brazil" )
+end
+
+local function runRandTp( caller, targets )
+    sendToPos( caller, targets, "#A randomly teleported #T" )
+end
+
+local brazilCommand = ulx.command( CATEGORY_TYPE, "ulx brazil", runBrazil, "!brazil" )
+brazilCommand:addParam{ type = ULib.cmds.PlayersArg }
+brazilCommand:defaultAccess( ULib.ACCESS_ADMIN )
+brazilCommand:help( "Sends target(s) to a random location on the map." )
+
+--Serious alias for brazil command
+local randTpCommand = ulx.command( CATEGORY_TYPE, "ulx randtp", runRandTp, "!randtp" )
+randTpCommand:addParam{ type = ULib.cmds.PlayersArg }
+randTpCommand:defaultAccess( ULib.ACCESS_ADMIN )
+randTpCommand:help( "Sends target(s) to a random location on the map." )
```

Comment:
ulx does allow you to have multiple chat commands but not multiple console commands, you could do that here but you may lose the ability to have different messages for the alias

this is how ulx teleport is defined using 2 chat commands
```lua
local teleport = ulx.command( CATEGORY_NAME, "ulx teleport", ulx.teleport, {"!tp", "!teleport"} )
```

im not going to say thats a required change but I thought id mention it

Reply:
ULX allows for multiple console commands if they're created through separate ulx.command() calls. Honestly up to Phatso if he wants there to be an alias still.